wandb/cli/cli.py

```diff
@@ -1185,9 +1273,20 @@ def launch_agent(
             "You must specify a project name or set WANDB_PROJECT environment variable."
         )
 
-    wandb.termlog("Starting launch agent ✨")
+    if sweep_id is not None:
```

Comment:
hmm i think if this is a short term hack so we have a valid entrypoint into the sweep agent, we should move it to the `wandb sweep-agent` command or make a new command. I don't want to pollute the launch-agent command with args that we may remove in the future since this is the code path we want to keep clean

Reply:
See above.